### PR TITLE
interop-RS documentation

### PR DESCRIPTION
**Pull request by runtologist**

### CODE REVIEW

File: microsite/src/main/tut/interop/reactivestreams.md [hunk 1]
```diff
@@ -6,23 +6,70 @@ title:  "Reactive Streams"
...
 **ZIO** integrates with [Reactive Streams](http://reactivestreams.org) by providing conversions from `zio.stream.Stream` to `org.reactivestreams.Publisher`
-and from `zio.stream.Sink` to `org.reactivestreams.Subscriber` and vice versa.
+and from `zio.stream.Sink` to `org.reactivestreams.Subscriber` and vice versa. Simply import `import scalaz.zio.interop.reactiveStreams._` to make the 
+conversions available.
 
-#### Example
+### Examples
 
+#### Publisher to Stream
 ```scala
```
Comment (iravid) [inline microsite/src/main/tut/interop/reactivestreams.md]: Can we add running examples using `tut`?

Comment (runtologist) [reply microsite/src/main/tut/interop/reactivestreams.md]: like so?

Comment (iravid) [reply microsite/src/main/tut/interop/reactivestreams.md]: Yup, thanks. How about running the streams in the examples as well?

**Comment by jdegoes** [comment]
@runtologist Ready to merge as soon as build is passing. Thanks for your work on this!

File: microsite/src/main/tut/interop/reactivestreams.md (hunk 1 above)
Comment (runtologist) [reply microsite/src/main/tut/interop/reactivestreams.md]: done. 
For Stream to Publisher we only see the unit result of subscribing the Subscriber to the Publisher. Making more visible here would require a lot of code around the actual business logic, awaiting a Promise, which is completed onComplete from the Subscriber. That would add a lot of noise around the actually pretty simpel code, so I left that out.

**Comment by runtologist** [comment]
btw, the build failure was due to an unrelated timeout. Maybe we can make the tests more tolerant to that?

**Comment by runtologist** [comment]
retest this please

**Comment by iravid** [comment]
restarted the build

**Comment by jdegoes** [comment]
@runtologist Thank you for your work on this!

**Pull request merged**